### ISSUE TITLE
Collection.configureRedisOplog causes allow/deny rules to run for server segregated mutations

### ISSUE THREAD

**Issue by copleykj** [cult-of-coders/redis-oplog#224]
I've run into an issue where I have trusted server side code that should not pass through allow/deny, are failing allow/deny checks.. If I comment out the call to configureRedisOplog for that collection everything runs fine.

**Comment by theodorDiaconu** [comment]
Do you mean that configureRedisOplog bypasses allow()/deny() ?

I don't see how this can be possible. Are you absolutely sure that triggers it ?
The configs are called only inside the mutations itself, not inside _validatedInsert/_validatedUpdate ...

**Comment by copleykj** [comment]
No, specifically I have an allow rule that is returning false when an insert is is performed from trusted code.

**Comment by theodorDiaconu** [comment]
Do you have a quick reproduction repo? If jot can u show me how configureRedisOplog looks? Thanks

Regards,
Theodor

**Comment by copleykj** [comment]
I'll see what I can do about getting one together this evening.

**Comment by copleykj** [comment]
So attempting a reproduction was unsuccessful which was baffling, so I spent the last 3 hours trying to figure out what this could possibly be. Turns out I was trying to access a property on an undefined selector inside of the mutation method passed to configureRedisOplog. For some reason though it wasn't throwing the error to the server console, but rather it was being sent back to the client and displayed as an access denied in the browser console. :confused:
